cime_config/machines/cmake_macros/intel_pm-cpu.cmake

```diff
@@ -29,3 +29,4 @@ string(APPEND CMAKE_Fortran_FLAGS " -fp-model=consistent -fimf-use-svml")
  string(APPEND CMAKE_Fortran_FLAGS_RELEASE " -g -traceback")
 string(APPEND CMAKE_Fortran_FLAGS " -DHAVE_ERF_INTRINSICS")
 string(APPEND CMAKE_CXX_FLAGS " -fp-model=consistent")
+set(MOAB_PATH "/global/cfs/cdirs/e3sm/software/moab/intel")
```

Comment:
What happens if you don't set MOAB_PATH?  Jim has said we shouldn't need both MOAB_PATH and MOAB_ROOT.

Reply:
update:  will remove these later.